kits/ai-experiment-planner/README.md

```
@@ -0,0 +1,168 @@
+---
+
+## ⚙️ Installation
+
+1. **Clone the repository**
+
+```bash
+git clone https://github.com/Lamatic/AgentKit.git
+cd AgentKit/kits/assistant/ai-career-copilot
+```
+
+2. **Install dependencies**
```

markdownlint-cli2 (0.23.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)